js/date-filter.js

```diff
@@ -25,8 +25,13 @@ function DateFilter(elm) {
     oncomplete: this.validate.bind(this)
   });
 
+  if (this.$minDate.attr('data-min-date')) {
```

Comment:
Is it really okay to have the date input fields empty? The page first loads with a one year period by default, so I think the fields should reflect that.